### PR TITLE
add quotes about hugs

### PR DESCRIPTION
**Pull request by practicatto**
Added 3 new quotes. Hope you like them :)

### CODE REVIEW

File: statuses.py [hunk 1]
```diff
@@ -58,4 +58,8 @@
    "Not good with words? Offer a hug. Hugs are the ultimate source of comfort.",
    "Be kind. Be lovable. Be huggable.",
    "Hugging is the silent way of saying \"You matter to me\".",
+   "No talk me, just hugs.",
```
Comment (achasveachas) [inline statuses.py:61]: The grammar here looks a bit off?

Maybe "No talk, just hugs." would be better?

File: statuses.py [hunk 2]
```diff
@@ -58,4 +58,8 @@
...
+   "Here's a hug for you from your favorite bot.",
+   "Virtual hugs can be just as good as real hugs.",
+
```
Comment (achasveachas) [inline statuses.py:64]: Nitpick: looks like you added an extra empty line by accident.

**Review by achasveachas — CHANGES_REQUESTED** [review]
These are great! 🤗 

 A few minor comments, once they are addressed I will merge.

File: statuses.py (hunk 1 above)
Comment (practicatto) [reply statuses.py:61]: Yes, it was intended because of the meme no talk me, im angy haha. But I'll change it :)

Comment (achasveachas) [reply statuses.py]: Ah, my bad, I wasn't familiar with that meme :)

**Review by achasveachas — APPROVED** [review]
Thanks 🤗 

If you have a Twitter handle let me know so I can give you a shout out! ❤️

**Comment by practicatto** [comment]
> 

I'm also practicatto in twitter c:

**Pull request merged**